internal/view/details.go

```diff
@@ -294,9 +295,12 @@ func (d *Details) saveCmd(evt *tcell.EventKey) *tcell.EventKey {
 
 func (d *Details) cpCmd(evt *tcell.EventKey) *tcell.EventKey {
 	d.app.Flash().Info("Content copied to clipboard...")
-	if err := clipboard.WriteAll(d.text.GetText(true)); err != nil {
+	if err := clipboard.Init(); err != nil {
 		d.app.Flash().Err(err)
 	}
+	if clipboard.Write(clipboard.FmtText, []byte(d.text.GetText(true))) == nil {
```

Comment:
if `Init` fail we will get panic here because clipboard is not initialized properly.